### ISSUE TITLE
MacOS scrool bar block close button

### ISSUE THREAD

**Issue by gagarine** [chzesa/tiled-tab-groups#7]
The scrool bar go on the way:
![image](https://user-images.githubusercontent.com/164074/56892348-aac5ee80-6a7f-11e9-97ad-3ddc19c2ddf1.png)

**Comment by chzesa** [comment]
Cannot reproduce on windows or ubuntu, and I don't have access to macos.

**Comment by fturcheti** [comment]
I don't think this is a bug of the extension, but a characteristic of this auto-hiding scrollbar. You just have to wait a second with the pointer outside the scrollbar area for it to disappear, so you can click the close button. Otherwise you can set the scrollbar to be always visible in the System Preferences.
